### DISCUSSION TITLE
Support - unable to via Paypal

### DISCUSSION THREAD

**Discussion by Deanobeano2** [cortexrd/Knack-Toolkit-Library#128] [Q&A]
Hi Norm

Is there another way we can help support you? Unfortunately, Paypal says "Donations to this recipient aren't supported in this country".

Dean

**Comment by cortexrd** [comment]
Really!?  What a bummer...
I'll to check about this and see if I can do something about it.
Will get back to you as soon as I've got something.

You can also check in your area (where are you located BTW?) if you can find a developer willing to collaborate on the KTL, and pay him/her directly.  Any money I'd receive will go to them anyways.

Cheers,
Norm

**Comment by Deanobeano2** [comment]
Hi Norm

 

I’m in New Zealand and I was also unable to pay Leigh Berrell in Australia for some code Sunny wrote that was beneficial to both of us. I had to resort to paying him via the bank.

 

Kind regards

Dean Morrison

General Manager
Excell Cleaning Services Limited

**Comment by cortexrd** [comment]
Hi Dean,

I've added a **Buy Me a Coffee** link.  Maybe I'll be luckier with this one?!

Cheers,
Norm

**Comment by Deanobeano2** [comment]
Hi Norm

 

That worked, thank you.

 

Kind regards

Dean Morrison

General Manager
Excell Cleaning Services Limited

**Comment by cortexrd** [comment]
Dean,

We've got it working now.
Your contribution is very appreciated, thanks a lot!!!

Norm
